tools/modules/system/manage_odroid_board.sh

```diff
@@ -0,0 +1,42 @@
+module_options+=(
+["manage_odroid_board,author"]=""
+["manage_odroid_board,ref_link"]=""
+["manage_odroid_board,feature"]="Odroid board"
+["manage_odroid_board,desc"]="Select optimised Odroid board configuration"
+["manage_odroid_board,example"]="manage_odroid_board"
+["manage_odroid_board,status"]="review"
+)
+#
+# @description Select optimised board configuration
+#
+function manage_odroid_board() {
+	local board_list=("Odroid XU4" "Odroid XU3" "Odroid XU3 Lite" "Odroid HC1/HC2")
```

Comment:
Keep indentation the same everywhere, similar to template
https://github.com/armbian/configng/blob/main/tools/template/module_software_template.sh

Also in module_options.